src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/webdriver/WebElementExt.kt

```diff
@@ -0,0 +1,32 @@
+package com.atlassian.performance.tools.jiraactions.api.webdriver
```

Comment:
`WebElementExt.kt` -> `HardenedKeyboard.kt`

This way it won't attract random behavior.